health-services/ExerciseSampleCompose/app/src/main/java/com/example/exercisesamplecompose/presentation/dialogs/ExerciseGoalMet.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.exercisesamplecompose.presentation.dialogs
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.SportsScore
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.wear.compose.material.Icon
+import androidx.wear.compose.ui.tooling.preview.WearPreviewDevices
+import com.example.exercisesamplecompose.R
+import com.google.android.horologist.compose.material.Confirmation
+
+@Composable
+fun ExerciseGoalMet(
+    showDialog: Boolean,
+) {
+    Confirmation(
+        modifier = Modifier
+            .fillMaxSize()
+            .fillMaxWidth(),
+        icon = { Icon(Icons.Default.SportsScore, contentDescription = "goal achieved") },
+        title = stringResource(id = R.string.goal_achieved),
+        showDialog = showDialog,
+        durationMillis = 10L,
```

Comment:
10ms seems pretty short, does this show for long enough?

Reply:
Agreed. I'll update it to 50, and maybe look into some sort of vibration in a later iteration. Definitely want to modify the ExerciseGoalMet component more because 1) it isn't centered and 2) it could look nicer 😅